.github/renovate.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["github>sapphiredev/readme:sapphire-renovate"],
+	"labels": ["Meta: Dependencies"],
```

Comment:
Won't labelsync cause this label to be outdated?

Reply:
Fixed